composer.json

```diff
@@ -8,7 +8,7 @@
     "minimum-stability": "beta",
     "prefer-stable": true,
     "require": {
-        "php": ">=5.5.9",
+        "php": "5.5.9  || ^7.0",
```

Comment:
This excludes 5.6, and newer versions of 5.5. I think this line was fine the way it was; the config you added below should do the trick without this change.

Reply:
Yup, that fixed it still too. Manually tested on php 7.3. Do we need some automated tests there? I've modified travis to do that just now. Let's see how that fairs.